.github/CODEOWNERS

```
@@ -182,9 +183,20 @@
 /pkg/config/system_probe.go             @DataDog/agent-network
 /pkg/config/remote/                     @DataDog/remote-config
 /pkg/config/remote/service/meta/        @DataDog/remote-config @DataDog/software-integrity-and-trust
+/pkg/diagnose/                          @Datadog/container-integrations
+/pkg/diagnose/connectivity/             @DataDog/agent-shared-components
+/pkg/epforwarder/                       @DataDog/agent-shared-components
+/pkg/flare/                             @DataDog/agent-shared-components
 /pkg/otlp/                              @DataDog/agent-platform
+/pkg/pidfile/                           @DataDog/agent-shared-components
+/pkg/persistentcache/                   @DataDog/agent-metrics-logs
+/pkg/proto/                             @DataDog/agent-shared-components
 /pkg/remoteconfig/                      @DataDog/remote-config
+/pkg/runtime/                           @DataDog/agent-shared-components
+/pkg/secrets/                           @DataDog/agent-shared-components
+/pkg/serializer/                        @DataDog/agent-shared-components
```

To me, this is tied to the aggregator, and more generally to metrics and logs (at least at the moment) so I'd define metrics-logs as the owner:

```suggestion
/pkg/serializer/                        @DataDog/agent-metrics-logs
```